### ISSUE TITLE
[release backmerge strategy] before starting a new release, validate some things in the previous release

### ISSUE THREAD

**Issue by pratul** [tramlinehq/tramline#144]
(this strategy is used by dunzo) 

1. before starting a new release, ensure previous release branch was merged into production branch 
2. before starting a new release, ensure working branch was backmerged into working branch 

We should also display both the merge PRs in the Finalize step of the older release.
